runner/database.go

```diff
@@ -344,6 +343,7 @@ func (ec EvalContext) EvalDatabasePanel(
 	pageIndex int,
 	panel *PanelInfo,
 	panelResultLoader func(projectId, panelId string) (chan map[string]any, error),
+	cache *CacheSettings,
```

Comment:
If you're using a pointer here you don't need DefaultCacheSettings. I'd pick one of them. nil is a default setting if you're using a pointer.